app/_src/gateway/kong-enterprise/secrets-management/advanced-usage.md

```diff
@@ -123,3 +123,13 @@ Parameter | UI field name | Description
 `vaults.description` *optional* | Description | An optional description for your vault.
 `vaults.name` | N/A | The type of vault. Accepts one of: `env`, `gcp`, `aws`, or `hcv`.
 `vaults.prefix` | Prefix | The reference prefix. You need this prefix to access secrets stored in this vault. For example, `{vault://my-env-vault/<some-secret>}`.
+
+Most of the vaults also support secret rotation by using TTLs:
+
+Parameter | Field name | Description
+----------|------------|------------
+`vaults.config.ttl` | **TTL** | Time-to-live (in seconds) of a secret from the vault when cached by this node.
```

Comment:
```suggestion
`vaults.config.ttl` | **TTL** | Time-to-live (in seconds) of a secret from the vault when cached.
```
"This node" seems to be ambiguous and I don't think it helps understanding the parameter better.